apps/frontend/src/components/global/groups/GroupManagement.vue

```diff
@@ -35,6 +35,51 @@
           />
         </v-container>
       </template>
+      <template #[`item.users`]="{item}">
+        <v-chip
+          :active="displayChip(0, 1, item.users)"
+          pill
+          color="primary"
+          class="ma-1"
+          v-on="on"
+          @click="displayMembersDialog(item.users[0])"
+        >
+          {{ getMemberNames(item.users)[0].identifier }}
+        </v-chip>
+        <v-chip
+          v-if="item.users.length >= 2"
+          :active="displayChip(992, 2, item.users)"
+          pill
+          color="primary"
+          class="ma-1"
+          v-on="on"
+          @click="displayMembersDialog(item.users[1])"
+        >
+          {{ getMemberNames(item.users)[1].identifier }}
+        </v-chip>
+        <v-chip
+          v-if="item.users.length >= 3"
+          :active="displayChip(1200, 3, item.users)"
+          pill
+          color="primary"
+          class="ma-1"
+          v-on="on"
+          @click="displayMembersDialog(item.users[2])"
+        >
+          {{ getMemberNames(item.users)[2].identifier }}
+        </v-chip>
```

Comment:
It should be possible to de-duplicate this by manipulating the data a little bit more before passing it to the component. If you have an array containing

`['Name 1', 'Name 2', 'Name 3', '+x total']`

then it should be possible to iterate it here instead of re-declaring the `v-chip` every time. There are a few examples where we do this in other parts of the application.